src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -109,8 +114,14 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
     public AddressBook load() throws StorageOperationException {
+        // create empty file if not found
```

Comment:
The purpose of this exercise is to check accidental file deletes and not check missing file initially. Therefore, the modification you have done in save() is the most appropriate one for the exercise.